src/core/client/admin/routes/Configure/ConfigureRoute.tsx

```diff
@@ -29,6 +30,11 @@ class ConfigureRoute extends React.Component<Props, State> {
     data: Parameters<Props["updateSettings"]>[0]["settings"],
     form: FormApi
   ) => {
+    // This ensures sites aren't saved to premoderateAllCommentsSites
+    // if the SINGLE_SITES_PRE moderation mode isn't selected
+    if (data.moderation !== GQLMODERATION_MODE.SINGLE_SITES_PRE) {
```

Comment:
Not a big deal, but for consistency sake, would mind if we renamed this  to `SINGLE_SITES_PREMOD`?

Reply:
I was going for consistency here with the `PRE` moderation mode. Are you thinking for consistency with the `PREMOD` status for comments?

Reply:
Oh, I see... my mistake. Yes, this is fine. I see the `PRE` and `POST` in there. We can keep this the way you have it!